less/site.less

```diff
@@ -97,6 +97,7 @@ map area {
       background-color: white;
       font-size: 0.9em;
       font-family: 'arial', sans-serif;
+      border: 1px solid black;
```

Comment:
the design spec provided showed this button without a border.. maybe we can have the words change color on hover instead?